app/Place.php

```diff
@@ -38,4 +38,13 @@ public function getCompleteAddressAttribute()
     {
         return "{$this->address}, {$this->city}, {$this->province}, {$this->postalCode}";
     }
+
+    /**
+     * Method returning the page title of the object
+     * @return string
+     */
+    public function getPageTitle(): string 
+    {
+        return $this->name . ' - Entreprise - ' . config('app.name', '');
```

Comment:
Same interogation as with categories.

Reply:
For Place, I would go  [Place name] - [Region] - [App name]. This way we would be SEO friendly with a Page Title not longer then 70 caracters.

Reply:
Done